docs/making-yomitan-dictionaries.md

```diff
@@ -0,0 +1,106 @@
+# Making Yomitan Dictionaries　<!-- omit in toc -->
+
+This document provides an overview on how to create your own Yomitan dictionary.
+
+- [Prerequisites](#prerequisites)
+- [General Process](#general-process)
+- [Tools](#tools)
+- [Read the Schemas](#read-the-schemas)
+- [Packaging A Dictionary](#packaging-a-dictionary)
+- [Examples](#examples)
+- [Schema Validation](#schema-validation)
+- [Conjugation](#conjugation)
+- [Tag Categories](#tag-categories)
+
+## Prerequisites
+
+A basic familiarity with JSON and some coding experience in a language of your choice is required.
+
+## General Process
+
+1. Acquire data (from a website, app, dump, etc.)
```

Comment:
"Acquire dictionary data" should be more clear

Reply:
How so? It's pretty obvious imo.